test/groovy/com/sap/piper/integration/TransportManagementServiceTest.groovy

```diff
@@ -66,7 +66,7 @@ class TransportManagementServiceTest extends BasePiperTest {
 
         assertThat(loggingRule.log, containsString("[TransportManagementService] OAuth Token retrieval started."))
         assertThat(loggingRule.log, containsString("[TransportManagementService] UAA-URL: '${uaaUrl}', ClientId: '${clientId}'"))
-        assertThat(loggingRule.log, containsString("Received response '{ \"access_token\": \"myOAuthToken\" }' with status 200."))
+        assertThat(loggingRule.log, containsString("Received response with status 200 from authentification request."))
```

Comment:
From my point of view, that is worth to check not only that the log contains some string, but also that it does not contain e.g. an ' \"access_token\": ' string.

Reply:
But we are not dealing with real responses / answers from a server anyway. We have on the level of mocking. And beside the log messages we emit ourselves to the log everything else is mocked. See some lines above `[content: 'access_token: "myOAuthToken"}'. What we put on the level of the mocks, we will receive and assert later.